--- a/Core/Util/PathUtil.cpp
+++ b/Core/Util/PathUtil.cpp
@@ -56,13 +56,13 @@ Path GetSysDirectory(PSPDirectories directoryType) {
 	case DIRECTORY_PSP:
 		return pspDirectory;
 	case DIRECTORY_CHEATS:
-		return pspDirectory / "Cheats";
+		return Path("/storage/roms/savestates/PPSSPPSDL/PSP/Cheats/");
 	case DIRECTORY_GAME:
-		return pspDirectory / "GAME";
+		return Path("/storage/roms/savestates/PPSSPPSDL/PSP/GAME/");
 	case DIRECTORY_SAVEDATA:
-		return pspDirectory / "SAVEDATA";
+		return Path("/storage/roms/savestates/PPSSPPSDL/PSP/SAVEDATA");
 	case DIRECTORY_SCREENSHOT:
-		return pspDirectory / "SCREENSHOT";
+		return Path("/storage/roms/screenshots");
 	case DIRECTORY_SYSTEM:
 		return pspDirectory / "SYSTEM";
 	case DIRECTORY_PAUTH:
